src/main/java/core/basesyntax/Square.java

```diff
@@ -0,0 +1,20 @@
+package core.basesyntax;
+
+public class Square extends AbstractFigure {
```

Comment:
The class name 'Square' is appropriate as it is a noun, but ensure that any interfaces you create do not use verbs for their names.